lib/src/components/NxDropdownMenu/NxDropdownMenu.scss

```diff
@@ -85,7 +85,11 @@
 
 .nx-dropdown-link {
   color: var(--nx-color-link);
+  margin: var(--nx-spacing-base) var(--nx-spacing-4x);
```

Comment:
Since doing this with margin is substantially more complicated, let's add a comment explaining why we're using margin.

Reply:
Updated here d671ceee7b4c09fe71af33dc1f2ee456af49a248.